docs/geospatial-install.md

```diff
@@ -1,29 +1,73 @@
+
+### Requirements
+Our [documentation](https://docs.knime.com/latest/business_hub_admin_guide/index.html#add-extension-docker-executor-images) describes the necessary requirements. Among them, one needs 
+- to enable the Execution Image Builder within the ```KOTS Admin Console``` > ```Config``` > ```Execution Image Builder``` > check ```Enable Execution Image Builder```
+- global admin priviliges
+- global admin application password
+
+## Manual Image Creation
+If it is desired or necessary to not to go with the Data App (e.g. due to an airgap installation or technical preference), one can do the following steps manually:
+- create the Dockerfile,
+- build an executor docker image,
+- push it to a registry, and eventually
+- make it available when creating or editing an execution context
+
+Besides the below steps, our [documentation](https://docs.knime.com/latest/business_hub_admin_guide/index.html#build-image-manually) also describes steps around manual docker image creation.
+
+### Requirements
 To be able to build your custom Docker image it is required to run [Docker](https://www.docker.com/), more precisely the [Docker Engine](https://docs.docker.com/engine/install/).
 Please make yourself familiar with the install instructions according to your operating system.
 In general, you can build the image locally. It is not required that the image is built somewhere on the KNIME Business Hub.
 
-## Dockerfile
+### Dockerfile
 The basis of a Docker image is a Dockerfile. It defines
-- the source image (in this case the official Docker image from KNIME; check out all available images [here](https://docs.knime.com/latest/business_hub_admin_guide/index.html#docker-executor-images)),
+- the source image (in this case the offical Docker image from KNIME; check out all available images [here](https://docs.knime.com/latest/business_hub_admin_guide/index.html#docker-executor-images)),
 - the libraries/packages to be installed, and subsequently 
 - which extensions should be installed into the executor.
 
 Make sure to save that file with the name ```Dockerfile``` and add no file extension.
 
 Within the Dockerfile, we define two variables at the bottom that are important while installing the extension(s):
 
-```KNIME_UPDATE_SITES``` - the URL to the necessary Update Site(s)
+```KNIME_UPDATE_SITES``` the URL to the necessary Update Site(s)
 
-```KNIME_FEATURES``` - the Feature ID of the extension to be installed
+```KNIME_FEATURES``` the Feature ID of the extension to be installed
 
-Check out [this video](https://www.youtube.com/watch?v=-dO79Id3VAo&t=143s) from the KNIMETV channel that helps you to find the right update site URL and the Feature ID.
+Check out [this video](https://www.youtube.com/watch?v=-dO79Id3VAo&t=143s) from the KNIMETV channel that helps you to find the right update site URL and the feature ID.
 
     # Define the base image
     FROM registry.hub.knime.com/knime/knime-full:r-5.2.1-369
```

Comment:
After release of 5.3.1, you can use from Docker hub directly and skip the registry.
> With the release of 5.3.2 the docker image name does not need the registry.hub.knime.com prefix anymore. Find more information about KNIME Executor images provided by KNIME here.
 use something like,
knime/knime-full:r-5.3.3-666